src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements BehaviorFigure {
```

Comment:
```suggestion
public class Circle extends Figure {
```
let Figure class implement interfaces, so all its children are also implementing interfaces